packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -2827,6 +2956,12 @@ msgstr "sazby DPH"
 msgid "VAT settings modified"
 msgstr "Nastavení DPH bylo upraveno"
 
+msgid "Valid from"
+msgstr ""
+
+msgid "Valid to"
+msgstr ""
```

Comment:
should this be translated too?

Reply:
c57add06f8

Reply:
Please translate also `Valid from` and `Bulk creation of promo codes` 😇

Reply:
:+1: 6e366d5817